Update the RDMS taxonomy to accommodate failover (#85)
finos/common-cloud-controls: services/database/relational/taxonomy.md

| CCC-RDMS-10  | Logging | Configurable logs are available for user inspection. |
| CCC-RDMS-11 | Monitoring | Configurable metrics are available for user inspection. |
| CCC-RDMS-12 | Alerting | Configurable alerts can be enabled. |

> | CCC-RDMS-13 | Failover | Standby database is trasitioned to the primary after the primary fails or become unreachable. |

Reviewer: ```suggestion
| CCC-RDMS-13 | Failover | Standby database can be implemented for failover when the primary can't be reached. |
```

Reviewer: consider rephrasing this to make it clear that this feature should be made available to users, rather than always automatically deployed (that would be part of the hardening controls)
